integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiJmsRecovery.java

```diff
@@ -0,0 +1,739 @@
+                                    .name(pvName)
+                                    .persistentVolumeClaim(new V1PersistentVolumeClaimVolumeSource()
+                                        .claimName(pvcName)))
+                            .addVolumeMountsItem(new V1VolumeMount()
+                                .mountPath("/shared")
+                                .name(pvName)))
+                    .adminServer(new AdminServer()
+                            .serverStartState("RUNNING")
+                            .adminService(new AdminService()
+                                    .addChannelsItem(new Channel()
+                                            .channelName("default")
+                                            .nodePort(0))))
+                    .addClustersItem(new Cluster()
+                            .clusterName("cluster-1")
+                            .replicas(replicaCount)
+                            .serverStartState("RUNNING"))
+                    .configuration(new Configuration()
+                            .secrets(securityList)
+                            .model(new Model()
+                                    .domainType("WLS")
+                                    .configMap(configmapName)
+                                    .runtimeEncryptionSecret(encryptionSecretName))
+                        .introspectorJobActiveDeadlineSeconds(300L)));
+    setPodAntiAffinity(domain);
+    logger.info("Create domain custom resource for domainUid {0} in namespace {1}",
+            domainUid, domNamespace);
+    boolean domCreated = assertDoesNotThrow(() -> createDomainCustomResource(domain),
+            String.format("Create domain custom resource failed with ApiException for %s in namespace %s",
+                    domainUid, domNamespace));
+    assertTrue(domCreated, String.format("Create domain custom resource failed with ApiException "
+                    + "for %s in namespace %s", domainUid, domNamespace));
+  }
+
+  private void checkPodNotCreated(String podName, String domainUid, String domNamespace) {
+    withStandardRetryPolicy
+        .conditionEvaluationListener(
+            condition -> logger.info("Waiting for pod {0} to be not created in namespace {1} "
+                    + "(elapsed time {2}ms, remaining time {3}ms)",
+                podName,
+                domNamespace,
+                condition.getElapsedTimeInMS(),
+                condition.getRemainingTimeInMS()))
+        .until(assertDoesNotThrow(() -> podDoesNotExist(podName, domainUid, domNamespace),
+            String.format("podDoesNotExist failed with ApiException for %s in namespace in %s",
+                podName, domNamespace)));
+  }
+
+  /*
+   * Verify the server MBEAN configuration through rest API.
+   * @param managedServer name of the managed server
+   * @returns true if MBEAN is found otherwise false
+   **/
+  private boolean checkManagedServerConfiguration(String managedServer) {
```

Comment:
looks like this is not used, can be removed

Reply:
removed